cookie-cutter-template/cookiecutter.json

```diff
@@ -0,0 +1,11 @@
+{
+  "repo_name": "Project",
+  "__repo_name": "{{cookiecutter.repo_name|lower|replace(' ', '_')}}",
```

Comment:
I think a somewhat common pattern is to call the normalized name(s) `<name>_slug` e.g. see here: https://github.com/Nicoretti/one-piece/blob/grand-line/python/python-cookie/cookiecutter.json